### PR TITLE
example with interactive exploration of properties obtained with regionprops

### PR DESCRIPTION
**Pull request by emmanuelle**
I added an example to the regionprops tutorial to illustrate how to visualize the properties directly on the figure.

image

### CODE REVIEW

**Comment by emmanuelle** [comment]
Please see the artifact https://653-2014929-gh.circle-artifacts.com/0/doc/build/html/auto_examples/segmentation/plot_regionprops.html#sphx-glr-auto-examples-segmentation-plot-regionprops-py when reviewing.

File: doc/examples/segmentation/plot_regionprops.py [hunk 1]
```diff
@@ -4,8 +4,8 @@
 =========================
 
 This example shows how to measure properties of labelled image regions. We
-analyze an image with two ellipses.
-
+first analyze an image with two ellipses. Below we show how to explore
+inteactively the properties of labelled objects.
```
Comment (emmanuelle) [inline doc/examples/segmentation/plot_regionprops.py:8]:
```suggestion
interactively the properties of labelled objects.
```

**Review by sciunto — APPROVED** [review]
so cool!

**Review by rfezzani — APPROVED** [review]
Cooool! I really love this demo :clap: I even didn't knew it was possible, thank you @emmanuelle .

**Review by jni — COMMENTED** [review]
This might be merged but I too want to signal my wholehearted approval! 😊

@VolkerH might also be interested in this as we were discussing making basically the same demo with napari. 😂

**Pull request merged**